api/test/integration/test_rootcheck_endpoints.tavern.yaml

```diff
@@ -495,77 +493,89 @@ stages:
           total_affected_items: 1
           total_failed_items: 0
 
-  # GET /rootcheck/001
-  - name: Check if data has been deleted in agent 001
+  # PUT /rootcheck
+  - name: Run rootcheck scan in all agents
     request:
       verify: False
-      method: GET
-      url: "{protocol:s}://{host:s}:{port:d}/rootcheck/001"
+      method: PUT
+      url: "{protocol:s}://{host:s}:{port:d}/rootcheck"
       headers:
         Authorization: "Bearer {test_login_token}"
     response:
       status_code: 200
       json:
-        error: 0
+        error: 2
         data:
-          affected_items: []
-          failed_items: []
-          total_affected_items: 0
-          total_failed_items: 0
-        message: !anystr
+          affected_items:
+            - "000"
+            - "001"
+            - "002"
+            - "003"
+            - "004"
+            - "005"
+            - "006"
+            - "007"
+            - "008"
+          failed_items:
+            - error:
+                code: 1601
+              id:
+                - "009"
+                - "010"
+            - error:
+                code: 1601
+              id:
+                - "011"
+                - "012"
+          total_affected_items: 9
+          total_failed_items: 4
 
-  # GET /rootcheck/002
-  - name: Check if agent 002 has data.
+
+---
+test_name: DELETE /rootcheck/{agent_id}
+
+stages:
+
+  # DELETE /rootcheck/001
+  - name: Delete rootcheck scans in agent 001
     request:
       verify: False
-      method: GET
-      url: "{protocol:s}://{host:s}:{port:d}/rootcheck/002"
+      method: DELETE
+      url: "{protocol:s}://{host:s}:{port:d}/rootcheck/001"
       headers:
         Authorization: "Bearer {test_login_token}"
     response:
       status_code: 200
       json:
         error: 0
         data:
-          affected_items: !anything
+          affected_items:
+            - "001"
           failed_items: []
-          total_affected_items: 5
+          total_affected_items: 1
           total_failed_items: 0
-        message: !anystr
 
-  # DELETE /rootcheck
-  - name: Try to delete rootcheck scans in all agents
+  # GET /rootcheck/001
+  - name: Check if data has been deleted in agent 001
```

Comment:
Add db delay after deleting and before reading results

Reply:
Done